extra/changelog-generator/test-changelog-generator

```diff
@@ -486,6 +589,9 @@ cat > expected.txt <<EOF
         dependency-type: direct:production
         update-type: version-update:semver-patch
       \`\`\`
+
+##### Other
```

Comment:
I'd like to discuss this one with you @kacf :smile:

Reply:
F2F?